unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarLODController/AvatarLODController.cs

```diff
@@ -6,33 +6,41 @@ namespace DCL
 {
     public interface IAvatarLODController : IDisposable
     {
-        void SetAvatarState();
+        Player player { get; }
+        void SetFullAvatar();
         void SetSimpleAvatar();
-        void SetImpostorState();
+        void SetImpostor();
+        void SetInvisible();
     }
 
     public class AvatarLODController : IAvatarLODController
```

Comment:
```suggestion
    public class AvatarLOD : IAvatarLOD
```
Please, remember to rename this in the future. Having `AvatarsLODController` and `AvatarLODController` with entirely different responsibilities is not ok. @pravusjif

Reply:
The only reason I havent done it yet is to avoid conflicts with al the on-going PRs we have. Gonna leave a comment so we don't forget